adr/2021_02_05_unified_api_prefix.md

```diff
@@ -56,7 +56,7 @@ header in addition to this path, however it may be misleading for consumers.
 - https://github.com/api-platform/core/issues/972
 - https://stackoverflow.com/questions/389169/best-practices-for-api-versioning
 - https://www.troyhunt.com/your-api-versioning-is-wrong-which-is/
-- http://codebetter.com/howarddierking/2012/11/09/versioning-restful-services/
+- https://codebetter.com/howarddierking/2012/11/09/versioning-restful-services/
```

Comment:
This link looks dead on both http and https.

Reply:
lets remove it then